src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -2123,6 +2123,39 @@ paths:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
+    delete:
+      x-alfresco-since: "7.1.1"
+      tags:
+        - renditions
+      summary: Delete rendition
+      description: |
+        **Note:** this endpoint is available in Alfresco 7.1.1 and newer versions.
+
+        Delete the rendition identified by **renditionId** of file **nodeId**.
```

Comment:
"file" does not sound right. Use what is used more generally to refer to nodeIds. I think you may have copied it from other 'renditions' descriptions. They look wrong too, but not suggesting fixing them at this time. I would suggest looking a the descriptions under the 'nodes' section.

Reply:
How about "Delete the rendition identified by **renditionId** of **nodeId**."?